x-pack/plugins/security_solution/public/common/components/page/use_refetch_by_session.tsx

```diff
@@ -57,10 +58,12 @@ export const useRefetchByRestartingSession = ({
   }, [dispatch, queryId, selectedInspectIndex]);
 
   useEffect(() => {
+    setSearchSessionId(session.current.start());
+    const currentSession = session.current;
     return () => {
-      data.search.session.clear();
```

Comment:
I updated this to use the `session.current` ref since its the only place we're not using the ref. I think it was done like this in the first place because of the following error:

```
ESLint: The ref value 'session.current' will likely have changed by the time this effect cleanup function runs. If this ref points to a node rendered by React, copy 'session.current' to a variable inside the effect, and use that variable in the cleanup function.(react-hooks/exhaustive-deps)
```
Thus the variable on L62